pkg/pillar/cas/containerd.go

```diff
@@ -993,6 +992,8 @@ func getIndexManifest(c *containerdCAS, blobSha256 string) (*v1.IndexManifest, e
 }
 
 // getManifestFromIndex: returns Manifest for the current architecture from IndexManifest
+//
+//nolint:unused
```

Comment:
IMO if there is unused function better to delete in a separate commit.

Reply:
I now removed this and some other unused code from cas.
@deitch was there any particular reason why you kept this code and fixed the linter warning rather than deleting the code?

Reply:
If I recall correctly, mostly because it was useful. We don't actually use it, and I really hoped one day we would just have a library for it, but it has useful generic functionality.